src/Quartz/CronExpression.cs

```diff
@@ -209,45 +209,68 @@ namespace Quartz
     [Serializable]
     public class CronExpression : IDeserializationCallback, ISerializable
     {
-        private static readonly Dictionary<string, int> monthMap = new Dictionary<string, int>(20);
-        private static readonly Dictionary<string, int> dayMap = new Dictionary<string, int>(60);
+        private static readonly IReadOnlyDictionary<string, int> monthMap = new Dictionary<string,int>(12)
```

Comment:
should use concrete types for private/internal members, otherwise there will unnecessary interface dispatch cost. new switch dispatch for string (NET7/NET8 compiler) might be faster here, not sure.

Reply:
Done. thanks for the performance advice!